ui_framework/components/collapse_button/collapse_button.js

```diff
@@ -0,0 +1,29 @@
+import React, {
+  PropTypes,
+} from 'react';
+import classNames from 'classnames';
+
+const DIRECTIONS = [
+  'down',
+  'up',
+  'left',
+  'right'
+];
+
+export const KuiCollapseButton = ({ className, direction, ...rest }) => {
+  const classes = classNames('kuiCollapseButton', className);
+  const childClasses = classNames('kuiIcon', `fa-chevron-circle-${direction}`);
+
+  return (<button
+            type="button"
+            className={classes}
+            {...rest}
+          >
+            <div className={childClasses}/>
+          </button>);
+};
+
+KuiCollapseButton.propTypes = {
+  className: PropTypes.string,
+  direction: PropTypes.oneOf(DIRECTIONS).isRequired
+};
```

Comment:
I think this is a case where we can export an object, like in the `button.js` file:

```javascript
export {
  DIRECTIONS,
  KuiCollapseButton,
};
```